src/main/resources/templates/client/hint/list.html

```diff
@@ -1,10 +1,91 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="cz" xmlns:th="http://www.w3.org/1999/xhtml">
```

Comment:
Jo, jinak sorry. To tam přidal kdysi já a už je to nakopírované všude. Ale správně je cs, ne cz.
Ale to já jen tak. Validátor to hlásí jako chybu potom.